get-started/inspect-traffic.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Inspecting Traffic
+slug: inspecting-traffic-get-started
+tags: inspecting web traffic, inspect sessions,analyzing captured traffic, Fiddler inspectors
+publish: true
+position: 50
+---
+
+## First Steps in Inspecting Web traffic
+
+With traffic already captured, the process of analyzing has begun. The first place where we can extract information about the captured sessions is the [**Web Sessions List**]({%slug web-sessions-list%}). Each session (a session is the request/response entry) is listed in a row with multiple column fields. You can extract useful information like the returned status code, the host and the URL, type of used protocol (HTTP vs. HTTPS), body sizer in bytes, used HTTP method (like GET or POST), and more (learn more about [the Web Sessions List fields]({%slug web-session-summary%}).
```

Comment:
1) The first sentence sounds strange. Capturing traffic does not sound the same as analyzing it. Do you mean that "the process of analyzing can begin" or something else?
2) "body sizer" -> "body size"?
3) Close the parenthesis (missing **)** ) or consider moving the content of the last parenthesis to a separate sentence:
"To learn more, refer to the Web Sessions List [the Web Sessions List fields]({%slug web-session-summary%})"